pkg/portfwd/client.go

```diff
@@ -29,7 +29,14 @@ func HandleTCPConnection(ctx context.Context, client *guestagentclient.GuestAgen
 	}
 
 	rw := &GrpcClientRW{stream: stream, id: id, addr: guestAddr, protocol: "tcp"}
-	bicopy.Bicopy(rw, conn, nil)
+
+	bicopy.Bicopy("tcp tunnel", bicopy.NamedReadWriter{
+		ReadWriter: rw,
+		Name:       guestAddr,
+	}, bicopy.NamedReadWriter{
+		ReadWriter: conn,
+		Name:       id,
```

Comment:
id is a description of the entire port forwading action: "tcp-%s-%s" (see line 17), so it is not a good for the conn.